src/test/model_cache_test.cpp

```diff
@@ -250,7 +250,7 @@ TEST_F(ModelCacheTest, PluginConfigChangeDoesImpactCache) {
 
     size_t lastCachedFileCount = currentCacheFileCount;
     currentCacheFileCount = this->getCachedFileCount();
-    ASSERT_GT(currentCacheFileCount, lastCachedFileCount);
+    ASSERT_GE(currentCacheFileCount, lastCachedFileCount);
```

Comment:
By changing only the assert from GT to GE the test name is wrong:
Plugin Config Change **Does** Impact Cache
I suggest to change it to ***DoesNot** and also change it to ASSERT_EQ to make sure it will never generate new files.